internal/querynode/impl.go

```diff
@@ -47,13 +47,11 @@ import (
 )
 
 // isHealthy checks if QueryNode is healthy
-func (node *QueryNode) isHealthy() bool {
-	code := node.stateCode.Load().(commonpb.StateCode)
+func (node *QueryNode) isHealthy(code commonpb.StateCode) bool {
```

Comment:
`isHealthy` and `isHealthyOrStopping` method seems no need to bind with querynode instance?

Reply:
True. Will more them into `common` package if any other component is reusing `Lifetime` component.